repo/packages/S/spark/13/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "packagingVersion": "2.0",
```

Comment:
In order to specify `.cli` in resources you must set `.packagingVersion` to `3.0`.